src/main/java/edu/cornell/kfs/sys/batch/service/impl/CuCacheServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package edu.cornell.kfs.sys.batch.service.impl;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.kuali.kfs.sys.batch.service.impl.CacheServiceImpl;
+import org.springframework.cache.CacheManager;
+
+public class CuCacheServiceImpl extends CacheServiceImpl {
+    private static final Logger LOG = LogManager.getLogger();
+
+    protected CacheManager cacheManager;
+
+    @Override
+    public void clearSystemCaches() {
+        LOG.debug("clearSystemCaches, cache manager class {}", cacheManager.getClass());
+        for (final String cacheName : cacheManager.getCacheNames()) {
+            LOG.debug("clearSystemCaches, attempting to clear cache for {}", cacheName);
```

Comment:
After looking at the spring-data-redis code further, you may want to also log the prefix info from the cache configuration object(s), since the prefixes might not necessarily be shown in the regular cache-name-returning methods.

Here are some examples of how to retrieve a cache's configuration object and the configured prefix. Note that you may have to typecast the cache manager as a `RedisCacheManager` (or typecast the returned cache as a `RedisCache`).

```java
cacheManager.getCacheConfigurationFor(cacheName).getKeyPrefix()
cacheManager.getCacheConfigurationFor(cacheName).getKeyPrefixFor(cacheName)
cacheManager.getCache(cacheName).getCacheConfiguration().getKeyPrefix()
cacheManager.getCache(cacheName).getCacheConfiguration().getKeyPrefixFor(cacheName)
```

Reply:
@jhulslander,

Sorry, I must have been looking at the wrong versions of the files for the first two calls. However, I believe the last two calls from earlier should work, if you typecast the `cacheManager.getCache()` result to a `RedisCache`:

```java
((RedisCache) cacheManager.getCache(cacheName)).getCacheConfiguration().getKeyPrefix()
((RedisCache) cacheManager.getCache(cacheName)).getCacheConfiguration().getKeyPrefixFor(cacheName)
```